core/src/test/java/apoc/cypher/CypherTest.java

```diff
@@ -66,12 +71,41 @@ public static void setUp() {
     public void clearDB() {
         db.executeTransactionally("MATCH (n) DETACH DELETE n");
         try (Transaction tx = db.beginTx()) {
-            tx.schema().getIndexes().forEach(IndexDefinition::drop);
             tx.schema().getConstraints().forEach(ConstraintDefinition::drop);
+            tx.schema().getIndexes().forEach(IndexDefinition::drop);
             tx.commit();
         }
     }
 
+    @Test
+    public void testDoIt() throws Exception {
```

Comment:
```suggestion
    public void testDoItAndRunWrite() throws Exception {
```

Reply:
What about parameterizing the test with a `functionName` parameter. The values would be `"doIt"` and `"runWrite"`. That way, you'd run the exact same test & assertions for both (as suggested by @JMHReif).
You could then get rid of the second half of this test.

Reply:
Imho it might not be worth it, because using the `@Parameterized.Parameters` annotation, all test would be executed twice. Instead with `@ParameterizedTest` in the single test, we should change all the test to make it compatible with junit5, so `@beforeClass`, `@ClassRule` etc. Maybe for 3 lines it's not worth it. However, I created a common function `runWriteAndDoItCommons` with this 2 use cases.

Reply:
Might be time to migrate to JUnit 5 😇